src/pages/settings/Profile/LoungeAccessPage.js

```diff
@@ -10,14 +12,35 @@ import useLocalize from '../../../hooks/useLocalize';
 import FeatureList from '../../../components/FeatureList';
 import IllustratedHeaderPageLayout from '../../../components/IllustratedHeaderPageLayout';
 import * as LottieAnimations from '../../../components/LottieAnimations';
+import compose from '../../../libs/compose';
+import withCurrentUserPersonalDetails, {withCurrentUserPersonalDetailsDefaultProps, withCurrentUserPersonalDetailsPropTypes} from '../../../components/withCurrentUserPersonalDetails';
+import LinearGradient from '../../../components/LinearGradient';
+import styles from '../../../styles/styles';
+import Avatar from '../../../components/Avatar';
+import Text from '../../../components/Text';
+import * as UserUtils from '../../../libs/UserUtils';
+import CONST from '../../../CONST';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import themeColors from '../../../styles/themes/default';
 
 const propTypes = {
+    /** The session of the logged in person */
+    session: PropTypes.shape({
+        /** Email of the logged in person */
+        email: PropTypes.string,
+    }),
+
     /** Current user details, which will hold whether or not they have Lounge Access */
     user: userPropTypes,
+
+    ...withLocalizePropTypes,
```

Comment:
Also remove these prop types after switching to `useLocalize` hook

Reply:
Done